pkg/osquery/tables/execparsers/apt/parser.go

```diff
@@ -0,0 +1,41 @@
+package apt
+
+import (
+	"bufio"
+	//"fmt"
+	"io"
+	"strings"
+)
+
+func aptParse(reader io.Reader) (any, error) {
+	results := make([]map[string]string, 0)
+
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := scanner.Text()
+		pair := strings.Split(line, "/")
+		if len(pair) < 2 {
+			continue
+		}
+
+		package_name := strings.ToLower(strings.TrimSpace(pair[0]))
+		if len(package_name) < 1 {
+			continue
+		}
+
+		values := strings.Split(pair[1], " ")
+		if len(values) < 6 {
+			continue
```

Comment:
Would we find it helpful to log a malformed line? Or do we expect to see a couple malformed lines already and that would just be noisy?

Reply:
I added a malformed test line